pkgs/top-level/all-packages.nix

```diff
@@ -10835,6 +10835,10 @@ with pkgs;
 
   nginxModules = callPackage ../servers/http/nginx/modules.nix { };
 
+  nginxShibboleth = callPackage ../servers/http/nginx/stable.nix {
+    modules = [ nginxModules.rtmp nginxModules.dav nginxModules.moreheaders nginxModules.shibboleth ];
+  };
```

Comment:
Do we have machinery that uses nginx dynamic modules in nixpkgs today? I too would like to use this and it seems unfortunate for all of us to have to compile nginx ourselves.